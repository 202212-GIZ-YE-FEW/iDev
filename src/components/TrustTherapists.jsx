import Image from "next/image";
import { withTranslation } from "next-i18next";

import Button from "./ui/Button";
function TrustTherapists({ t }) {
    return (
        <>
            <div className='container flex flex-col space-y-8 justify-between'>
                <p className='text-3xl md:text-4xl lg:text-5xl uppercase break-words'>
                    {t("title of trust therapists")}.
                </p>
                <Image
                    src='/trust_therapists.png'
                    alt='trust_therapist'
                    width={600}
                    height={300}
                />
                <p className='pb-20'>{t("description of trust therapists")}.</p>
                <div className='w-fit'>
                    <Button
                        content={t("book an appointment")}
                        text-transform='uppercase'
                        filled='true'
                        size='large'
                        fontSize='text-lg md:text-xl lg:text-2xl'
                        radius='md'
                    />
                </div>
            </div>
        </>
    );
}
export default withTranslation("home")(TrustTherapists);
