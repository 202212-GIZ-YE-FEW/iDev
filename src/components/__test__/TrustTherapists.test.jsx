import renderer from "react-test-renderer";

import TrustTherapists from "../TrustTherapists";

it("renders correctly", () => {
    const tree = renderer.create(<TrustTherapists />).toJSON();
    expect(tree).toMatchSnapshot();
});
