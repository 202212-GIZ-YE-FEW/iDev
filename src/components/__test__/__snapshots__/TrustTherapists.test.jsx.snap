// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-yellow py-16"
>
  <div
    className="container flex flex-col space-y-8 justify-between"
  >
    <p
      className="text-3xl md:text-4xl lg:text-5xl uppercase break-words"
    >
      title of trust therapists
      .
    </p>
    <img
      alt="trust_therapist"
      data-nimg="1"
      decoding="async"
      height={300}
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      src="/_next/image?url=%2Ftrust_therapists.png&w=1200&q=75"
      srcSet="/_next/image?url=%2Ftrust_therapists.png&w=640&q=75 1x, /_next/image?url=%2Ftrust_therapists.png&w=1200&q=75 2x"
      style={
        Object {
          "color": "transparent",
        }
      }
      width={600}
    />
    <p
      className="pb-20"
    >
      description of trust therapists
      .
    </p>
    <div
      className="w-fit"
    >
      <button
        className="font-normal whitespace-nowrap cursor-pointer capitalize text-lg md:text-xl lg:text-2xl rounded-md px-[25px] md:px-[35px] lg:px-[59px] py-[12px] font-medium bg-cyan text-black "
      >
        book an appointment
      </button>
    </div>
  </div>
</div>
`;
